docs/source/reference/server-api.md

```diff
@@ -302,21 +262,20 @@ DELETE /hub/api/users/:user/servers/[:servername]
 
 **Required scope: `servers`**
 
-Like start, delete may not complete immediately.
-The DELETE request has two possible response codes:
+Similar to when starting a server, issuing the DELETE request above might not stop the server immediately. Instead, the DELETE request has two possible response codes:
 
 204 Deleted
 : This status code means the delete completed and the server is fully stopped.
 It will now be absent from the user `servers` model.
 
 202 Accepted
-: Like start, `202` means your request was accepted,
-but is not yet complete.
+: This code means your request was accepted, but is not yet completely processed.
 The server has `pending: 'stop'` at this point.
 
-Unlike start, there is no progress API for stop.
-To wait for stop to finish, you must poll the user model
-and wait for the server to disappear from the user `servers` model.
+There is no progress API for checking when a server actually stops.
+Thus, the only available alternative is to poll the server and wait for it to disappear from the user `servers` model.
+
+This Python code snippet can be used to check if a server stops:
```

Comment:
That's not quite what the code does. It _stops_ the server, and then waits for it to finish stopping.

Reply:
I have rephrased the sentence. Please check whether it is accurate.